sda-download/api/sda/sda_test.go

```diff
@@ -677,7 +677,8 @@ func (f *fakeGRPC) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	_, err = w.Write([]byte{0})
 	assert.NoError(f.t, err, "Could not write response flag")
 
-	err = binary.Write(w, binary.BigEndian, int32(len(response)))
+	assert.Less(f.t, len(response), int(^uint32(0)), "Response too long")
```

Comment:
I might be confusing things here, but why not use `math.MaxInt32` instead? ` int(^uint32(0)=4294967295` and `int32(4294967294)`  does overflow..?

Reply:
1. I didn't think to use a built in function for this!
2. I didn't notice that here we deal with `int32` and NOT `unint32` (see e.g. the fix for reencrypt) :blush: Great catch!

My old self would be inclined to change `int(^uint32(0))` to `int(^uint32(0)/2)` and marvel at the fact that no new module was imported for this bit. 
But my new self values readability so this is now changed, thanks for pointing out :smile: